packages/next-app/src/components/Dialog.tsx

```diff
@@ -29,8 +31,9 @@ export function Dialog({
         />
         <DialogPrimitive.Content
           className={cn(
-            "data-[state=open]:animate-contentShow fixed top-[50%] left-[50%] max-h-[85vh] translate-x-[-50%] translate-y-[-50%] rounded-[6px] bg-blue3 p-[25px] shadow-[hsl(206_22%_7%_/_35%)_0px_10px_38px_-10px,_hsl(206_22%_7%_/_20%)_0px_10px_20px_-15px] focus:outline-none  overflow-y-auto",
-            customWidth ? customWidth : "w-[90vw] max-w-[450px]"
+            "data-[state=open]:animate-contentShow fixed top-[50%] left-[50%] max-h-[85vh] translate-x-[-50%] translate-y-[-50%] rounded-[6px] bg-blue3 shadow-[hsl(206_22%_7%_/_35%)_0px_10px_38px_-10px,_hsl(206_22%_7%_/_20%)_0px_10px_20px_-15px] focus:outline-none",
+            customWidth ? customWidth : "w-[90vw] max-w-[450px]",
```

Comment:
could also be 
```suggestion
            { "w-[90vw] max-w-[450px]": customWidth },
```
but I'd argue that if you're taking in `customWidth` I'd probably expect you to pass that width right? 😬

Reply:
yes I'm using like this 

```js
   <Dialog
      content={<TokenModal handleSelectToken={handleSelectToken} />}
      customWidth="w-8/12"
      noPadding
    >

```

Reply:
🤦 read this too fast, feel free to ignore